src/Particular.PlatformSample.ServicePulse/Particular.PlatformSample.ServicePulse.csproj

```diff
@@ -1,28 +1,32 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net48</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
     <IncludeBuildOutput>false</IncludeBuildOutput>
     <SuppressDependenciesWhenPacking>true</SuppressDependenciesWhenPacking>
     <Description>Particular ServicePulse binaries for use by Particular.PlatformSample. Not intended for use outside of Particular.PlatformSample.</Description>
```

Comment:
I know it's outside the changeset, but does it still make sense to refer to the "binaries" here?

Reply:
If anything, I'd say that's more correct than it was before. Previously we were just packaging all the JS files, but now we're actually shipping the host binaries.